locales/en.json

```diff
@@ -1038,6 +1038,9 @@
     "walletAnswer": "Wallets are used to send, receive, store, and display digital assets",
     "walletLink": "Learn about wallets",
     "walletHeading": "Connect Wallet",
+    "walletDetails": "Your Wallet",
+    "walletAccount": "My {0} Account",
```

Comment:
Are we sure about this wording? `My roileo Account` feels a bit weird

Reply:
Actually, it would show the wallet name, according to the design.

<img width="212" alt="image" src="https://user-images.githubusercontent.com/31397967/215532038-b55d85b4-7a22-4148-bddf-1df25f11a012.png">

Reply:
In this example isn't it "My Polkadot Account" the full wallet name?

Reply:
I mean `wallet extension name`, like polkadot-js \ metamask \ talisman.

Maybe @exezbcz  could tell us which name should be display : )

Reply:
Oops my bad I didn't realize that was extension name and not identity